### PR TITLE
Add Eastern Michigan University

### PR DESCRIPTION
**Pull request by programmer94**
Added 2FA info for Eastern Michigan University (EMU). Many thanks to the IT department for providing the icon!

### CODE REVIEW

**Review by Carlgo11 — DISMISSED** [review]
Looking good! 👍

**Comment by RichJeanes** [comment]
Perhaps this should have an exception stating that it is for University employees, not students?
>Duo is licensed now for all EMU employees (including student employees, staff, and faculty). 
-- https://www.emich.edu/it/security/esp/duo.php

**Comment by programmer94** [comment]
I can add that, sure.

File: _data/education.yml [hunk 1]
```diff
@@ -127,6 +127,16 @@ websites:
...
       doc: https://www.ecu.edu/cs-itcs/office365/multi-factor-authentication.cfm
 
+    - name: Eastern Michigan University
+      url: https://www.emich.edu
+      img: emu.png
+      tfa: Yes
+      hardware: Yes
+      software: Yes
+      exceptions:
+          text: "Available for university employees, not students."
```
Comment (stephengroat) [inline _data/education.yml]: probably best to mirror their language (since student employees are actually valid)?

what about `"Available for employees"`

**Review by stephengroat — CHANGES_REQUESTED** [review]

File: _data/education.yml (hunk 1 above)
Comment (conorgil) [reply _data/education.yml]: @programmer94 just a friendly ping to get this top of inbox for you. Sounds like this PR is good to go with just a few last tweaks

**Review by stephengroat — APPROVED** [review]

**Pull request merged**